doc/integrator/headers.rst

```diff
@@ -30,6 +36,58 @@ Where ``<path>`` can be: ``main``, ``admin``, ``apihelp`` or ``c2c``,
 ``connect_src`` or ``worker_src``,
 ``[_extra]`` is a suffix to be able to extend a directive instance of completely overriding it.
 
+Access-Control-Allow-Origin
+---------------------------
+
+To add domain that can access to the authenticated routes you should have something like that
+in the vars file:
+
+.. code:: diff
+
+    headers:
+      dynamic: &header {}
+      index: *header
+      api: *header
+      apihelp: *header
+      profile: *header
+      raster: *header
+      error: *header
+  -   themes: &auth_header {}
+  +   themes: &auth_header
+  +     access_control_allow_origin:
+  +         - "{VISIBLE_WEB_PROTOCOL}://{VISIBLE_WEB_HOST}"
+  +         - "https://your.custom.domain"
+  +         - "*"
+      config: *auth_header
+      print: *auth_header
+      fulltextsearch: *auth_header
+      mapserver: *auth_header
+      tinyows: *auth_header
+      layers: *auth_header
+      shortener: *auth_header
+  -   login: *auth_header
+  +   login:
+  +     access_control_allow_origin:
+  +         - "{VISIBLE_WEB_PROTOCOL}://{VISIBLE_WEB_HOST}"
+  +         - "https://your.custom.domain"
+
+Global headers
+--------------
+
+For path the not manage with the ``headers`` section we can use the ``global_headers`` section
```

Comment:
I am not sure what we want to say here. Is it this:
"For paths not managed in the headers section, a global headers section can be defined in the vars file with the following syntax:"